cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/PipelineTriggersUtils.java

```diff
@@ -31,34 +31,45 @@
 import io.cdap.cdap.etl.proto.v2.TriggeringPropertyMapping;
 import io.cdap.cdap.internal.io.SchemaTypeAdapter;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.lang.reflect.Type;
 import java.util.List;
 import java.util.Map;
 
-public final class WorkflowScheduleUtils {
+/**
+ * Handles parsing of pipeline runtime arguments mapping.
+ */
+public final class PipelineTriggersUtils {
```

Comment:
nit: we normally don't put 'Utils' in the name, just PipelineTriggers.

Reply:
Aha, I followed the other files naming convention, will update